test/regressions/webpack.config.js

```diff
@@ -32,12 +32,11 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.(js|ts|tsx)$/,
-        // prism.js blocks @mui/internal-markdown/prism from being interpreted as ESM in this build.
-        exclude: /node_modules|prism\.js/,
+        test: /\.(js|mjs|ts|tsx)$/,
```

Comment:
If I understand corectly, this is to be able to run test on markdown new files.

I've seen similar update on the mui-x PR (https://github.com/mui/mui-x/pull/16112) but no .mjs files added. Is their a specific reason?

Reply:
Yes, I added it there as well, only this rule is configured in webpackBaseConfig for some reason. After I move the tests to vite, I think only eslint will still be using this config for its aliases, so we should be able to remove the rule altogether.